core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -71,8 +71,12 @@ object ApiVersion {
     "0.11.0-IV2" -> KAFKA_0_11_0_IV2,
     "0.11.0" -> KAFKA_0_11_0_IV2,
     // Introduced LeaderAndIsrRequest V1, UpdateMetadataRequest V4 and FetchRequest V6 via KIP-112
+    // 0.11.1 was never released, this version only existed in trunk until we decided that 1.0.0 would be the feature
+    // release after 0.11.0.x
     "0.11.1-IV0" -> KAFKA_0_11_1_IV0,
-    "0.11.1" -> KAFKA_0_11_1_IV0
+    "0.11.1" -> KAFKA_0_11_1_IV0,
```

Comment:
@becketqin @lindong28, it would be nice if we could remove 0.11.1-IV0 and 0.11.1. If LinkedIn is not relying on that yet, then it's probably safe (I doubt anyone else would be using trunk since the JBOD KIP was merged). Let us know.